bip-musig2/reference.py

```diff
@@ -352,6 +355,29 @@ def test_key_agg_vectors():
     assert key_agg([X[0], X[0], X[0]], [], []) == expected[2]
     assert key_agg([X[0], X[0], X[1], X[1]], [], []) == expected[3]
 
+
+def test_nonce_gen_vectors():
+    def fill(i):
+        return (i).to_bytes(1,byteorder='big')*32
```

Comment:
```suggestion
        return i.to_bytes(1, byteorder='big') * 32
```

Reply:
I think the parantheses around `i` are required.

Reply:
Ok, may depend on the python version, it works for me without. Let's keep it. 

May still be good to add the spacing but that's really a micronit.